server/pages/index.jade

```diff
@@ -1,135 +0,0 @@
-doctype html
-//
```

Comment:
Another option is to pass it as part of `express-react-view` `doctype` option, since it's simply concatenated anyways:

https://github.com/reactjs/express-react-views/blob/c200d55/index.js#L50-L56